apps/backend/src/graphql/services/account.ts

```diff
@@ -8,62 +9,104 @@ import {
   TeamUser,
   User,
 } from "@/database/models/index.js";
+import { transaction } from "@/database/transaction.js";
+import { uninstallSlackInstallation } from "@/slack/index.js";
 import { cancelStripeSubscription } from "@/stripe/index.js";
 
-import { deleteProject } from "./project.js";
+import { unsafe_deleteProject } from "./project.js";
 
-export const getAdminAccount = async (args: {
+/**
+ * Get an account by ID, ensuring the user has admin permissions.
+ */
+export async function getAdminAccount(args: {
   id: string;
   user: User | undefined | null;
-}): Promise<Account> => {
+  withGraphFetched?: RelationExpression<Account>;
+}): Promise<Account> {
   invariant(args.user, "no user");
-  const account = await Account.query().findById(args.id).throwIfNotFound();
+  const query = Account.query().findById(args.id).throwIfNotFound();
+  if (args.withGraphFetched) {
+    query.withGraphFetched(args.withGraphFetched);
+  }
+  const account = await query;
   const permissions = await account.$getPermissions(args.user);
   invariant(permissions.includes("admin"), "not admin");
   return account;
-};
+}
 
-export const deleteAccount = async (args: {
+/**
+ * Delete an account and all associated projects and subscriptions.
+ */
+export async function deleteAccount(args: {
   id: string;
   user: User | undefined | null;
-}) => {
+}): Promise<void> {
   const account = await getAdminAccount({
     id: args.id,
     user: args.user,
+    withGraphFetched: "[projects,slackInstallation]",
```

Comment:
It would be better to provide a list instead of a string here

Reply:
I prefer the string syntax notation